.pre-commit-config.yaml

```
@@ -4,11 +4,11 @@ repos:
   - repo: https://github.com/ambv/black
     rev: 22.1.0
     hooks:
-    - id: black
-      args:
-        - --safe
-        - --quiet
-      files: ^((esphome|script|tests)/.+)?[^/]+\.py$
+      - id: black
+        args:
+          - --safe
+          - --quiet
+        files: ^((esphome|script|tests)/.+)?[^/]+\.py$
```

Please change in its own PR.

```suggestion
    - id: black
      args:
        - --safe
        - --quiet
      files: ^((esphome|script|tests)/.+)?[^/]+\.py$
```